pkg/volume/local_volume/local_volume.go

```diff
@@ -251,10 +280,24 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
-	// TODO: how to prevent multiple mounts with conflicting fsGroup?
 	return volume.SetVolumeOwnership(m, fsGroup)
 }
 
+// isBindMounted checks whether the volume has beend mounted before.
+func (m *localVolumeMounter) isBindMounted() (bool, error) {
+	mountpoints, err := m.mounter.List()
+	if err != nil {
+		return false, err
+	}
+	for _, mountpoint := range mountpoints {
```

Comment:
Yes, this can be large, that's why i'm checking "fsGroup != nil" at first to avoid unnecessary mount lookup, based on the observation that 

Regarding the cache, I was actually searching for such cache but with no success. I did a quick bench to see how long it takes.

```
for i in `seq 350`; do docker run -itd nginx:1.13; done
```

This gives me roughly 1000+ mounts. Then creating a pod with fsGroup (multiple times), gives the following numbers for running isBindMounted:

```
running time 30.906658ms
running time 49.563007ms
running time 62.251977ms
running time 56.930919ms
running time 37.011615ms
```

I'm not sure if this is acceptable in kubelet, and kubelet is fairly complex now, should we involve more people here?